docs/ban_nested_it.md

```diff
@@ -0,0 +1,44 @@
+## Expect not to have nested 'it' call expressions
+
+Nested `it` call expression in spec files will cause false positives for unit tests.
+
+To fix this, the the upper `it` call expression should be changed to a `describe`.
+
+### Postive Examples:
```

Comment:
positive misspelled

Also it would be awesome if we used the executable test cases to generate this section of the docs (I did that for Error Prone) - for later :)

Reply:
fix the spelling